tv_apps/app-deck/build/build.js

```diff
@@ -4,10 +4,25 @@
 
 var utils = require('utils');
 
+var AppDeckBuilder = function() {};
+
+// XXX: we should extract the logic of copying default-cards.json
+// to one common place. See https://bugzil.la/1123585
+AppDeckBuilder.prototype.execute = function(options) {
+  var stageDir = utils.getFile(options.STAGE_APP_DIR);
+  var configFile = utils.getFile(stageDir.path, 'js', 'init.json');
```

Comment:
it is generated and copied to `build_stage/` by this script. It will be loaded in each app as card list for first time start-up .